iocore/net/AcceptOptions.h

```diff
@@ -25,52 +25,54 @@
 #pragma once
 
 #include "tscore/ink_inet.h"
+
 #include "I_Event.h"
+#include "I_Net.h"
 
 struct AcceptOptions {
   using self = AcceptOptions; ///< Self reference type.
 
   /// Port on which to listen.
   /// 0 => don't care, which is useful if the socket is already bound.
-  int local_port;
+  int local_port = 0;
   /// Local address to bind for accept.
   /// If not set -> any address.
   IpAddr local_ip;
   /// IP address family.
   /// @note Ignored if an explicit incoming address is set in the
   /// the configuration (@c local_ip). If neither is set IPv4 is used.
-  int ip_family;
+  int ip_family = AF_INET;
   /// Should we use accept threads? If so, how many?
-  int accept_threads;
+  int accept_threads = -1;
   /// Event type to generate on accept.
-  EventType etype;
+  EventType etype = ET_NET;
```

Comment:
I think you could remove this (and not have to include "I_Net.h") because it is never set to anything other than ET_NET.

Reply:
Yes, I don't see it being set to anything else.  You could get rid of the member variable and just use ET_NET in the code.